nextflow.config

```diff
@@ -137,6 +137,9 @@ params {
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
+// Load Sentieon config
+includeConfig 'conf/sentieon.config'
```

Comment:
It seems to me that we that, you wouldn't be able to run anything without having `SENTIEON_AUTH_MECH_BASE64` and `SENTIEON_AUTH_DATA_BASE64` set as nf-secrets.

Reply:
No, that is actually the problem. By referring to those two secrets in `cache.config` without them being set, Ram got

`Unknown config secret 'SENTIEON_AUTH_MECH_BASE64' `

Reply:
You write too fast 😆  "No, that is actually the problem." was my answer to "How so? if these aren't set, surely it just ignores them?".

This 
```
ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64 ?: ''
```
seems to work. In fact, it could probably just have been introduced in the orginial `cache.config` and then problem solved.

Reply:
Hmmm - I didn't know about that "License Server Extension". It is not something we use at my workplace. Perhaps make an issue for making Sarek handle that as well?

Reply:
This just in from Ram: He still gets
```
Unknown config secret 'SENTIEON_AUTH_MECH_BASE64' -- Check script 
```
when using
```
        ext.sentieon_auth_mech_base64 = secrets.SENTIEON_AUTH_MECH_BASE64 ?: ''
        ext.sentieon_auth_data_base64 = secrets.SENTIEON_AUTH_DATA_BASE64 ?: ''
```
in `cache.config` and commercial-license (ie. only SENTIEON_LICENSE_BASE64 as nf-secret)